picking_dispatch/picking_dispatch.py

```
@@ -331,9 +331,27 @@ def _get_related_dispatch(self, cr, uid, ids, field_names, arg=None,
             result[picking_id].append(dispatch_id)
         return result
 
+    def _search_dispatch_pickings(self, cr, uid, obj, name, args,
+                                  context=None):
+        dispatch_ids = []
+        picking_ids = []
+        for arg in args:
+            if isinstance(arg, tuple):
+                dispatch_ids = arg[2]
+        if dispatch_ids:
+            move_obj = self.pool['stock.move']
+            move_ids = move_obj.search(cr, uid,
+                                       [('dispatch_id', 'in', dispatch_ids)],
```

Any operator can be used when searching, not only `in`. I think they should not all translate to the domain `('dispatch_id', 'in', dispatch_ids)`. Some operators such as '<', '>' should raise an error (use a whitelist of allowed operators). `not in` or `=` should directly be used in the domain instead of `in`.